core/src/main/java/org/apache/brooklyn/util/core/logbook/BrooklynLogEntry.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.brooklyn.util.core.logbook;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.brooklyn.util.exceptions.Exceptions;
+
+import java.util.Date;
+
+/**
+ * This class models the log format used by Apache Brooklyn by default
+ * @see `etc/org.ops4j.pax.logging.cfg`
+ */
+public class BrooklynLogEntry {
+
+    @JsonProperty("timestamp")
+    String timestampString;
+    Date datetime;
```

Comment:
what is relationship between `datetime` and `timestampString` ?  worth a comment here?

Reply:
@ahgittin `timestampString` is the value coming from the elasticSearch data base, but `datatime` is this time transformed to a Date datatype in the method 'FileLogStore#parseLogLine' using the format you can provide as a property on `brooklyn.cfg` that depends on the log configuration.

It has a default value so no new properties needed if the log config is not changed